scipy/sparse/linalg/isolve/tests/test_minres.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import division, print_function, absolute_import
+
+import numpy as np
+from numpy.testing import (assert_equal, assert_allclose, assert_,
+                           run_module_suite)
```

Comment:
The run_module_suite should not be imported or used in this file.

Reply:
Got it, updating now.